chihacknight/decarbonize-my-state
# Start FAQ Page, Mostly with EV Questions

## Overview

Adds an FAQ page and links it in the header - this page answers a lot of the common questions I have gotten particularly about electric vehicles, and I plan on adding more info about renewable energy.

Starts on issue #30 but doesn't close it out yet. I'm opening this PR now since there's already a lot of text that might take us some time to discuss, but I think this starter FAQ is better than not having one.

### Demo

Check out the Netlify - this is mostly a giant text page

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

- Load the Netlify and confirm you can go to `/faq` from the header
- Confirm the page is readable on desktop and mobile
- Actually read all the text.
